docs/docusaurus/docs/guides/connecting_to_your_data/fluent/filesystem/how_to_connect_to_one_or_more_files_using_pandas.md

```diff
@@ -42,7 +42,9 @@ import AfterCreateNonSqlDatasource from '/docs/components/connect_to_data/next_s
 
 ### 1. Import the Great Expectations module and instantiate a Data Context
 
-<ImportGxAndInstantiateADataContext />
```

Comment:
Is there a reason why this was removed? I believe the component is compatible with the current script + it includes some additional text as well. Currently this section is just the code, without explanation. 

We could either 
1. Keep the original `ImportGxAndInstantiateADataContext` if  it works. 
2. Add the `The code to import Great Expectations and instantiate a Data Context is:` text from the component to this doc

Reply:
@Shinnnyshinshin I used Option 1 (this is consistent with what I had done with other similar situations).  Good catch -- thank you,